sources/assets/exegol/skel/bloodhound_customqueries/README.txt

```diff
@@ -0,0 +1,11 @@
+If you wish to have your own "Bloodhound customqueries.json" file, you are at the right place.
```

Comment:
I think the folder can just be named "bloodhound", no need to add complexity here imo

Reply:
I renamed the folder to `bloodhound` as suggested.
However, as you rightfully requested the support for the config.json file. I thought it was not clear without the distinction.
For this reason, the sub folder `merge` and `replace` were renamed `customqueries_merge`.